components/support/base/src/main/java/mozilla/components/support/base/android/locale/Extensions.kt

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.base.android.locale
```

Comment:
Let's move this into a separate component! :)

Reply:
👍

Reply:
Moved to `support-locale`.